src/components/sidebar/routePathView/routePathListTab/RoutePathListLink.tsx

```diff
@@ -1,22 +1,74 @@
 import * as React from 'react';
-import { IRoutePathLink } from '~/models';
+import { observer, inject } from 'mobx-react';
+import { FiChevronRight } from 'react-icons/fi';
+import { IRoutePathLink, INode } from '~/models';
+import { RoutePathStore } from '~/stores/routePathStore';
+import { Button } from '~/components/controls';
+import ButtonType from '~/enums/buttonType';
 import RoutePathListObject, { ListObjectType } from './RoutePathListObject';
+import InputContainer from '../../InputContainer';
+import * as s from './routePathListObject.scss';
 
 interface IRoutePathListLinkProps {
+    routePathStore?: RoutePathStore;
+    previousNode?: INode;
     routePathLink: IRoutePathLink;
+    nextNode?: INode;
 }
 
+@observer
+@inject('routePathStore')
```

Comment:
Switch order of these decorators.
`observer` should be applied first and now the decorators would translate to
`observer(inject('routePathStore)(RoutePathListLink))`

Reply:
This issue also appears in _RouteLayer.tsx_